File: 5KYU/Josephus Survivor/solution.java
public class JosephusSurvivor {
  public static int josephusSurvivor(final int n, final int k) {
      int r = 0;
      int i = 1;
    
      while(i != n+1) {
         r = (r + k) % i;
         i++;
      }
      return r+1;
  }  
}

File: 5KYU/Josephus Survivor/solution.cs
using System;
public class JosephusSurvivor 
{
  	public static int JosSurvivor(int n, int k) 
    {
    	  var r = 0;
        var i = 1;
      
        while(i != n+1)
        {
          r = (r + k) % i;
          i++;
        }
      
        return r+1;
    }  
}